pkg/cmd/util/log.go

```diff
@@ -27,3 +29,15 @@ func (w *gLogWriter) Write(p []byte) (n int, err error) {
 
 	return len(p), nil
 }
+
+// InitLogrus sets the logrus trace level based on the glog trace level.
+func InitLogrus() {
+	switch {
+	case bool(glog.V(5)):
```

Comment:
Our five is super noisy. I think i see our levels as

 1. Normal - info
 2. normal with limited, but useful debug - 2
 3. debug with useful output for developers to solve a problem-   4
 4. debug with absolutely everything we have that will explode your logs - 8

mapping those here would put 
 1. info at  V(2)
 2. debug at V(4)

Reply:
this is why it would be nice to set it independently. :)